### PR TITLE
NMS-15027: Add JSON support to the Kafka producer for flows

### PR DESCRIPTION
**Pull request by j-white**
JIRA: https://issues.opennms.org/browse/NMS-15027

Allows JSON encoding to be enabled with:
```
config:edit org.opennms.features.flows.persistence.kafka
config:property-set useJson true
config:update
```

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/flows/setup.adoc [hunk 1]
```diff
@@ -204,8 +204,9 @@ Set to `false` to disable statistic recording.
 
 Enriched flows (with OpenNMS node data) can also be forwarded to Kafka.
 
-Enriched flows are stored in `flowDocuments` topic and the payloads are encoded using link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers (GPB)].
+Enriched flows are stored in `flowDocuments` topic and the payloads are encoded using link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers (GPB)] by default.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/setup.adoc:207]:
```suggestion
Enriched flows are stored in `flowDocuments` topic and the payloads are encoded using link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers] (GPB) by default.
```

File: docs/modules/operation/pages/deep-dive/flows/setup.adoc [hunk 2]
```diff
@@ -204,8 +204,9 @@ Set to `false` to disable statistic recording.
...
 See `flowdocument.proto` in the corresponding source distribution for the model definitions.
+JSON support can be enabled by setting the `useJson` property to `true`.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/setup.adoc:209]:
```suggestion
To enable JSON support, set the `useJson` property to `true`.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor changes in the docs. Thanks!

**Review by emarsh1995 — COMMENTED** [review]
I've got nothing to add. Once Bonnie's changes are made, I'll approve the docs. Thanks!

**Review by Bonrob2 — DISMISSED** [review]
LGTM!

**Review by emarsh1995 — DISMISSED** [review]
Docs LGTM. Thank you!

**Review by cgorantla — APPROVED** [review]
LGTM!

**Pull request merged**